src/tests/end-to-end/common/page.ts

```diff
@@ -18,9 +18,10 @@ export class Page {
         underlyingPage.on('pageerror', error => {
             forceEventFailure(`'pageerror' (console.error) with stack: ${error.stack}`);
         });
-        underlyingPage.on('requestfailed', request => {
-            forceEventFailure(`'requestfailed' from '${request.url()}' with errorText: ${request.failure().errorText}`);
-        });
+        // Removing this error handler as a temporary workaround for #923
+        // underlyingPage.on('requestfailed', request => {
+        //     forceTestFailure(`request failed - ${request.failure().errorText}, ${request.url()}`);
```

Comment:
Consider checking the URL (maybe for prefix `https://static2.sharepointonline.com`, or suffix `.woff2`) and not calling forceTestFailure if it matches.

Reply:
Updated to check for `fonts` and `icons` in the URL